lib/elixir/lib/task/supervisor.ex

```diff
@@ -298,17 +308,19 @@ defmodule Task.Supervisor do
   Similar to `start_child/2` except the task is specified
   by the given `module`, `fun` and `args`.
   """
-  @spec start_child(Supervisor.supervisor, module, atom, [term]) :: {:ok, pid}
+  @spec start_child(Supervisor.supervisor(), module, atom, [term]) :: {:ok, pid}
   def start_child(supervisor, module, fun, args) when is_atom(fun) and is_list(args) do
     Supervisor.start_child(supervisor, [get_info(self()), {module, fun, args}])
   end
 
   defp get_info(self) do
-    {node(),
-     case Process.info(self, :registered_name) do
-       {:registered_name, []} -> self
-       {:registered_name, name} -> name
-     end}
+    {
+      node(),
+      case Process.info(self, :registered_name) do
+        {:registered_name, []} -> self
+        {:registered_name, name} -> name
+      end
+    }
```

Comment:
This is exactly the stuff that we want contributors to refactor, as outlined in the first example in
https://github.com/elixir-lang/elixir/issues/6643. :)

In this case,

```elixir
name =
  case Process.info(self, :registered_name) do
    {:registered_name, []} -> self
    {:registered_name, name} -> name
  end

{node(), name}
```